CHANGELOG.md

```diff
@@ -21,6 +38,11 @@ All notable changes to this project will be documented in this file. The format
   - Raise awareness in the troubleshooting page for a common misconfiguration when deviating from our advice by using a bare domain ([#3680](https://github.com/docker-mailserver/docker-mailserver/pull/3680))
 - **Internal:**
   - Postfix configures `virtual_mailbox_maps` and `virtual_transport` during startup instead of using defaults (configured for Dovecot) via our `main.cf` ([#3681](https://github.com/docker-mailserver/docker-mailserver/pull/3681))
+  - Updates to `packages.sh`:
+    - The script now uses `/etc/os-release` to determine the release name of Debian
+    - Removed custom installations of Fail2Ban, getmail6 and Rspamd
+    - Updated packages lists and added comments for maintainability
+    - Use official Rspamd PPA on AMD64; no PPA is used on ARM64
```

Comment:
This line isn't accurate anymore? Both ARM64 and AMD64 are used?

```suggestion
    - Use official Rspamd PPA
```

Reply:
```suggestion
    - Use official Rspamd PPA for AMD64 and ARM64
```